draft-ietf-acme-acme.md

```diff
@@ -662,10 +660,10 @@ contact (optional, array of string):
 related to this authorization. For example, the server may wish to notify the
 client about server-initiated revocation.
 
-agreement (optional, string):
-: A URI referring to a subscriber agreement or terms of service provided by the
-server (see below).  Including this field indicates the client's agreement with
-the referenced terms.
+terms-of-service-agreed (optional, boolean):
+: Including this field in a new-registration request, with a value of true,
+indicates the client's agreement with the terms of service. This field is not
+updateable by the client.
```

Comment:
Summarizing offline comment: Even though this seems inconsistent, we're going to only support initial agreement in-band, not re-agreement.  The rare CA that needs re-agreement can use the mechanism we're in #182.